### PR TITLE
Conversations: improve caterpillar label

### PR DESCRIPTION
**Pull request by bluefuton**
From a usability point of view, I think there are two problems with the current caterpillar label:
- it implies that there are _x_ more authors to expand, not _x_ more comments
- it does not suggest that you can interact with the caterpillar to expand those comments

I've attempted to address both in this PR. Feedback welcome!

Before:

screen shot 2017-12-07 at 11 18 57
screen shot 2017-12-07 at 11 19 06

After:

screen shot 2017-12-07 at 11 19 29
screen shot 2017-12-07 at 11 19 18

### To test

Visit http://calypso.localhost:3000/read/conversations.

### CODE REVIEW

File: client/blocks/conversation-caterpillar/index.jsx [hunk 1]
```diff
@@ -119,33 +120,39 @@ class ConversationCaterpillarComponent extends React.Component {
...
+					{ commentCount > 1 &&
+						uniqueAuthorsCount === 1 &&
+						translate( '%(count)d more comments from %(commenterName)s', {
+							args: {
+								commenterName: lastAuthorName,
+								count: commentCount,
+							},
+						} ) }
+					{ commentCount === 1 &&
+						translate( '1 more comment from %(commenterName)s', {
```
Comment (a8ci18n) [inline client/blocks/conversation-caterpillar/index.jsx]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'View comment from %(commenterName)s' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/conversation-caterpillar/index.jsx [hunk 2]
```diff
@@ -119,33 +120,39 @@ class ConversationCaterpillarComponent extends React.Component {
...
+						uniqueAuthorsCount > 1 &&
+						translate( '%(count)d more comments from %(commenterName)s and others', {
+							args: {
+								commenterName: lastAuthorName,
+								count: commentCount,
+							},
+						} ) }
+					{ commentCount > 1 &&
+						uniqueAuthorsCount === 1 &&
+						translate( '%(count)d more comments from %(commenterName)s', {
```
Comment (a8ci18n) [inline client/blocks/conversation-caterpillar/index.jsx]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'View comments from %(commenterName)s and %(count)d more' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/caterpillar-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/conversation-caterpillar/index.jsx [hunk 3]
```diff
@@ -119,33 +120,39 @@ class ConversationCaterpillarComponent extends React.Component {
...
-								},
-							} )
-						: translate( '%(commenterName)s commented', {
-								args: {
-									commenterName: lastAuthorName,
-								},
-							} ) }
+					{ commentCount > 1 &&
+						uniqueAuthorsCount > 1 &&
+						translate( '%(count)d more comments from %(commenterName)s and others', {
```
Comment (a8ci18n) [inline client/blocks/conversation-caterpillar/index.jsx]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'View comments from %(commenterName)s and %(count)d more' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/caterpillar-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/conversation-caterpillar/index.jsx [hunk 4]
```diff
@@ -119,33 +120,39 @@ class ConversationCaterpillarComponent extends React.Component {
...
-									},
-								} )
-							: translate( 'View comment from %(commenterName)s', {
-									args: {
-										commenterName: lastAuthorName,
-									},
-								} )
-					}
+					title={ translate(
+						'View %(count)s more comments for this post',
```
Comment (blowery) [inline client/blocks/conversation-caterpillar/index.jsx]: i think these are backwards (see the bottom of https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/caterpillar-copy)

**Comment by jancavan** [comment]
@bluefuton Should we remove "more"?

File: client/blocks/conversation-caterpillar/index.jsx (hunk 4 above)
Comment (bluefuton) [reply client/blocks/conversation-caterpillar/index.jsx]: Good spot! Fixed.

**Comment by michelleweber** [comment]
Yeah, I would delete "more." Otherwise good!

File: client/blocks/conversation-caterpillar/index.jsx [hunk 5]
```diff
@@ -119,33 +120,39 @@ class ConversationCaterpillarComponent extends React.Component {
...
+					{ commentCount > 1 &&
+						uniqueAuthorsCount === 1 &&
+						translate( '%(count)d comments from %(commenterName)s', {
+							args: {
+								commenterName: lastAuthorName,
+								count: commentCount,
+							},
+						} ) }
+					{ commentCount === 1 &&
+						translate( '1 comment from %(commenterName)s', {
```
Comment (a8ci18n) [inline client/blocks/conversation-caterpillar/index.jsx]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'View comment from %(commenterName)s' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/caterpillar-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/conversation-caterpillar/index.jsx [hunk 6]
```diff
@@ -119,33 +120,39 @@ class ConversationCaterpillarComponent extends React.Component {
...
+						uniqueAuthorsCount > 1 &&
+						translate( '%(count)d comments from %(commenterName)s and others', {
+							args: {
+								commenterName: lastAuthorName,
+								count: commentCount,
+							},
+						} ) }
+					{ commentCount > 1 &&
+						uniqueAuthorsCount === 1 &&
+						translate( '%(count)d comments from %(commenterName)s', {
```
Comment (a8ci18n) [inline client/blocks/conversation-caterpillar/index.jsx]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'View comments from %(commenterName)s and %(count)d more' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/blocks/conversation-caterpillar/index.jsx [hunk 7]
```diff
@@ -119,33 +120,39 @@ class ConversationCaterpillarComponent extends React.Component {
...
-								},
-							} )
-						: translate( '%(commenterName)s commented', {
-								args: {
-									commenterName: lastAuthorName,
-								},
-							} ) }
+					{ commentCount > 1 &&
+						uniqueAuthorsCount > 1 &&
+						translate( '%(count)d comments from %(commenterName)s and others', {
```
Comment (a8ci18n) [inline client/blocks/conversation-caterpillar/index.jsx]: Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'View comments from %(commenterName)s and %(count)d more' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/reader/caterpillar-copy)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by jancavan** [comment]
Nice work @bluefuton!

**Comment by blowery** [comment]
woohoo! thanks @bluefuton !

**Pull request merged**